examples/metrics-ingestion-otel/README.md

```diff
@@ -0,0 +1,26 @@
+# DataPrepper Metrics Ingestion from OpenTelemetry Collector
+
+This is an example of using the OpenTelemetry Collector to send metrics data to Data Prepper and then to OpenSearch.
+The Data Prepper OTLP/gRPC endpoint is exposed at port 21891. 
+The same protocol can be used with the OpenTelemetry Collector, which listens at the OTLP default port 4317.
+This setup allows to compare both endpoints.
+The Collector will forward any data to Data Prepper for indexing in OpenSearch.
+
+To generate some demo data, the OpenTelemetry Collector uses its host metrics receiver to acquire cpu and memory metrics on the machine it is running on.
+Additionally, it scrapes the Prometheus metrics endpoint of the Data Prepper instance.
+This also let's you investigate the Data Prepper metrics in OpenSearch.
+
+To run:
+
+1. Run `docker compose up`
+2. Wait for everything to come up.
+3. Log into OpenSearch Dashboards at <http://localhost:5601>.
+4. Create an Index Pattern for index `otel_metrics` choosing `time` as the time field.
```

Comment:
Should the pattern by `otel_metrics*`?

Reply:
I deliberately chose not to use a roll-over index. Therefore, no wildcard is rerquired. The idea was, to keep the example as simple as possible. Do you want to change it to a daily index?